drivers/retained_mem/retained_mem_sf32lb_rtc_bkp.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2025, Qingsong Gou <gouqs@hotmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sifli_sf32lb_rtc_backup
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/drivers/retained_mem.h>
+
+#include <string.h>
+
+LOG_MODULE_REGISTER(retained_mem_sf32lb_rtc_bkp, CONFIG_RETAINED_MEM_LOG_LEVEL);
+
+struct retained_mem_sf32lb_config {
+	uintptr_t base;
+	uintptr_t sub_base;
+	size_t size;
+};
+
+static ssize_t retained_mem_sf32lb_size(const struct device *dev)
+{
+	const struct retained_mem_sf32lb_config *cfg = dev->config;
+
+	return (ssize_t)cfg->size;
+}
+
+static int retained_mem_sf32lb_read(const struct device *dev, off_t offset, uint8_t *buf,
+					size_t len)
```

Comment:
the second lines of split functions are not aligned (use spaces **after** tabs to align properly)